PathPlanning/FrenetOptimalTrajectory/path_planner.py

```diff
@@ -0,0 +1,563 @@
+"""
```

Comment:
Almost code same as frenet_optimal_trajectory.py. Could you please modify this code to use the function of frenet_optimal_trajectory.py?

Reply:
Sorry, maybe I misunderstood your comment. Do you mean that I have to extend frenet_optimal_trajectory.py, or can I keep a separated script which uses the same functions? Because for a two vehicles simulation I need to parse a json file, which contains obstacles and a reference trajectory.

Reply:
I mean  you can keep a separated script which uses frenet_optimal_trajecty's function. is it difficult?

Reply:
It's not difficult, but in order to implement _Follow Mode_ and _Low velocity_ trajectory generation I have to change a bit **calc_frenet_paths**, so it's like having three different function, one for each modality.

Reply:
Check out last commit.